docs/docs/use-cases/keda.md

```diff
@@ -0,0 +1,201 @@
+---
+comments: true
+---
+
+# Scaling Workloads with KEDA based on Keptn metrics
+
+If you want to use [KEDA](https://keda.sh/) to scale your workloads, you can set it up
+to consume metrics from Keptn.
+This gives you great flexibility for your setup since Keptn can
+consolidate different observability providers for you and KEDA
+will simplify the scaling operations for you.
+
+This use case is enabled by the Keptn Metrics
+[custom API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
+that the Keptn Metrics Operator provides.
+
+## Before you begin
+
+For this use case presentation, a few components need to be deployed to the
+cluster in order to have a full setup:
+
+- Keptn Metrics Operator: Will be used for metrics consolidation.
+
+    For more information about installation please refer to the
+    [installation guide](../installation/index.md).
+
+    > The Keptn Lifecycle Operator does not need to be installed for this use-case.
+
+- [KEDA](https://keda.sh/): Will be used for scaling.
+- [Prometheus](https://prometheus.io/): Will be used as metrics provider.
+  
+    For more information about how to install Prometheus into your cluster, please
+    refer to the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation/).
```

Comment:
Should menion that they also need a `KeptnMetricsProvider` resource defined.  Maybe a reference to the "Keptn Metrics" guide chapter?

Reply:
the provider will set up later in the page and is not a prerequisite